src/backend/commands/explain_gp.c

```diff
@@ -1492,6 +1492,21 @@ cdbexplain_showExecStatsBegin(struct QueryDesc *queryDesc,
 	return ctx;
 }								/* cdbexplain_showExecStatsBegin */
 
+/*
+ * cdbexplain_showStatCtxFree
+ *	  Release memory allocated for CdbExplain_ShowStatCtx structure and its
+ *	  internals.
+ */
+void
+cdbexplain_showStatCtxFree(struct CdbExplain_ShowStatCtx *ctx)
+{
+	Assert(ctx != NULL);
+
+	pfree(ctx->extratextbuf.data);
+	pfree(ctx->slices);
```

Comment:
@andr-sokolov, can you explain why should we not free memory allocated for members of CdbExplain_SliceSummary structure? (e.g. for workers, memoryAccounts, memoryAccountCount)? If you rely on execution `ExplainPrintPlan` in the short-living query memory context, please reword the comment above.

Reply:
I think we should complete free CdbExplain_SliceSummary structure before `pfree(ctx->slices)`

Reply:
> @andr-sokolov, can you explain why should we not free memory allocated for members of CdbExplain_SliceSummary structure? (e.g. for workers, memoryAccounts, memoryAccountCount)? If you rely on execution `ExplainPrintPlan` in the short-living query memory context, please reword the comment above.

Done